src/backend/settings.py

```diff
@@ -515,10 +515,6 @@
     ("css", "compressor_toolkit.precompilers.SCSSCompiler"),
 )
 
-#: Command that will be executed to transform ES6 into ES5 code (see
-#: `django-compressor-toolkit <https://github.com/kottenator/django-compressor-toolkit>`__)
-COMPRESS_ES6_COMPILER_CMD = 'export NODE_PATH="{paths}" && {browserify_bin} "{infile}" -o "{outfile}" -t [ "{node_modules}/babelify" --presets [ @babel/preset-env ] --plugins [ @babel/plugin-transform-runtime ] ]'
```

Comment:
I think you can also remove all compressor-related settings as well as remove "compressor" and "compressor_toolkit" from `INSTALLED_APPS` and the Pipfile. This might fix the `bundle-static-files` job of the CircleCI workflow...

Reply:
The CI issue was a JS file I missed while merging. That should be fixed. I also removed compress and related code in my last commit. CI seems to be happy now 👍🏽